src/Models/BulkDragBoundaryConditions.jl

```diff
@@ -0,0 +1,298 @@
+"""
+    BulkDragBoundaryConditions
+
+Module for implementing quadratic bottom drag boundary conditions for velocity fields.
+
+Provides `BulkDragFunction` for computing bottom momentum fluxes using bulk aerodynamic
+formulas. The drag function computes a quadratic drag:
+
+```math
+τᵘ = - Cᴰ |U| u
+```
+
+where `Cᴰ` is the drag coefficient and `|U| = √(u² + v²)` is the horizontal speed.
+
+This is the velocity flux analog of Breeze.jl's `BulkDragFunction`, which computes
+momentum flux `Jᵘ = -Cᴰ |U| ρu`.
+"""
```

Comment:
If this hardcodes a quadratic dependence, then I think it should have `Quadratic` in the name. After all there are non-quadratic bulk drags.

Furthermore, this assumes a _bottom_ drag. This is fine but it's large-scale-GFD-centric. Is it hard to make it more general? Even in some oceanographic applications (with steep bathymetry) lateral drag can be important. If you wanna keep it as is maybe call it `QuadraticBottomBulkDragBoundaryCondition`? (A mouthful, I know...)

Reply:
"bulk formula" based on similarity theory are always quadratic

Reply:
> "bulk formula" based on similarity theory are always quadratic

Agreed, for similarity theory. But bulk formulations don't need to come from similarity theory, no?

Reply:
@tomchor if you are interested, I can open a PR after this that does some generalization, and we can work on validation together.